reaper/cogs/log_reading.py

```diff
@@ -432,17 +432,19 @@ async def on_message(self, message):
                     value="Please note that I am a bot and am still heavily being worked on. There is a chance that some or all of this information is incorrect, in which case I apologize.\nIf you still encounter issues after doing this, please send another log.",
                     inline=False,
                 )
-                await message.channel.send(embed=problem, reference=message, view=view)
+                await message.reply(embed=problem, reference=message, view=view)
```

Comment:
The change to `reply` seems to break it something so I'm reverting it ^^"

Traceback (most recent call last):
  File "Reaper/.venv/lib/python3.12/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "Reaper/reaper/cogs/log_reading.py", line 435, in on_message
    await message.reply(embed=problem, reference=message, view=view)
  File "Reaper/.venv/lib/python3.12/site-packages/discord/message.py", line 1564, in reply
    return await self.channel.send(content, reference=self, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: discord.abc.Messageable.send() got multiple values for keyword argument 'reference'


```suggestion
                await message.channel.send(embed=problem, reference=message, view=view)
```